package.json

```diff
@@ -155,6 +155,6 @@
     "underscore": "1.8.3"
   },
   "optionalDependencies": {
-    "fsevents": "*"
+    "fsevents": "^2.3.1"
```

Comment:
@jasleennuno-okta  do we need this ? I am able to run//instlal without this. However  if it is fixing the fsevent issues seems like a good addition. 

I am worried about other changes to yarn.lock file. /cc @haishengwu-okta

Reply:
what's current issue?

Reply:
i was having an issue with yarn install yesterday
i removed the change, removed node_modules, and ran yarn install again and succeeded
so i removed the change from the PR entirely